exercises/takudzwa_makoni/ex02_oo_basics/src/isoTriangle.h

```diff
@@ -8,12 +8,12 @@ class isoTriangle : public shape { // isosceles triangle class
 private:
 
 	static std::string m_name;
```

Comment:
Unless you want the name to change, i would make this `const` too.

Reply:
replaced with member function shapeType()